slack.md

```diff
@@ -14,6 +14,9 @@ Once you've joined, there are a number of channels, depending on what you are lo
 
 - **#general**: Questions/ideas/links of general interest to the community. When in doubt, post here.
 - **#events**: Announcements/discussion of events that might be interesting to the group.
+- **#python-en-español**: Hablar de Python y las charlas en español
```

Comment:
While I'm not a native speaker, this doesn't ring exactly right with me. One obvious inconsistency is that the other channels are identified using noun phrases characterizing the sorts of posts to be found in the channel, while this is a verb phrase. More importantly, the phrase itself does not feel like it's very idiomatic. 

Perhaps we should run this by a native speaker before we merge it?

Reply:
We could say "discusión en español sobre Python."

Reply:
Thanks Jose, I made that change in 034287a